lib/parliament-flow.js

```diff
@@ -29,7 +35,33 @@ WHERE {
 }`;
   const response = await query(queryString);
   const parsed = parseSparqlResults(response)?.[0];
-  return parsed ?? { parliamentFlow: null, parliamentSubcase: null };
+  return { 
+    parliamentFlow: null, 
+    parliamentSubcase: null,
+    ...parsed
+  };
+}
+
+/**
+ * @returns {Promise<ParliamentFlow[]>}
```

Comment:
**chore**: Add the param as well, which would be something like `@param {string[]} statuses`

Reply:
fixed in [8edb714](https://github.com/kanselarij-vlaanderen/vlaams-parlement-sync-service/pull/7/commits/8edb7144d5ca03ce7a2ca682235a771b7b75056e)